From 13e165388b6f9fe70252adbb4a2ee0cf5ccdbb86 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Wed, 10 Apr 2024 00:27:16 +0200
Subject: [PATCH 03/10] scripts/t/mk: replace double quotes with single quote
 in shell recipes

This escapes more characters for the shell, and makes the intent of
the dollar character more explicit.
---
 scripts/t/mk/architecture.mk | 2 +-
 scripts/t/mk/buildflags.mk   | 2 +-
 scripts/t/mk/buildtools.mk   | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/t/mk/architecture.mk b/scripts/t/mk/architecture.mk
index b146f34b7..b181d3426 100644
--- a/scripts/t/mk/architecture.mk
+++ b/scripts/t/mk/architecture.mk
@@ -19,4 +19,4 @@ loop_targets := $(foreach machine,BUILD HOST TARGET,$(foreach var,$(vars),\
 test: $(loop_targets)
 
 $(loop_targets):
-	test "$($@)" = "$(TEST_$@)"
+	test '$($@)' = '$(TEST_$@)'
diff --git a/scripts/t/mk/buildflags.mk b/scripts/t/mk/buildflags.mk
index 3289a7557..042ba816f 100644
--- a/scripts/t/mk/buildflags.mk
+++ b/scripts/t/mk/buildflags.mk
@@ -23,4 +23,4 @@ loop_targets := $(vars) $(vars:=_FOR_BUILD)
 test: $(loop_targets)
 
 $(loop_targets):
-	test "$($@)" = "$(TEST_$@)"
+	test '$($@)' = '$(TEST_$@)'
diff --git a/scripts/t/mk/buildtools.mk b/scripts/t/mk/buildtools.mk
index c104ef62e..2b935ebb1 100644
--- a/scripts/t/mk/buildtools.mk
+++ b/scripts/t/mk/buildtools.mk
@@ -22,4 +22,4 @@ loop_targets := $(tools) $(tools:=_FOR_BUILD)
 test: $(loop_targets)
 
 $(loop_targets):
-	test "$($@)" = "$(TEST_$@)"
+	test '$($@)' = '$(TEST_$@)'
-- 
2.39.2

